ambry-tools/src/main/java/com.github.ambry/store/ConsistencyCheckerTool.java

```diff
@@ -347,17 +348,17 @@ private long getLogOffsetEnd(File indexFile)
       }
       return fileEndPointer;
     }
+  }
 
-    class IndexFileNameComparator implements Comparator {
-      public int compare(Object o1, Object o2) {
-        String s1 = ((File) o1).getName();
-        String s2 = ((File) o2).getName();
+  static class IndexFileNameComparator implements Comparator {
```

Comment:
are you going to do all the "reuse store code" changes in another commit? Because all of this functionality is already available and this is repeated code.